SAMLPortal/wwwroot/js/site.js

```diff
@@ -2,3 +2,47 @@
 // for details on configuring this project to bundle and minify static web assets.
 
 // Write your JavaScript code.
+function getResultsFromFilters()
+{
+	let uri = "/api/setup/getResultsFromFilters";
+	let results = [];
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['results' is defined but never used.](https://app.codacy.com/manual/zegorax/SAMLPortal/pullRequest?prid=5343548)